index.js

```diff
@@ -11,40 +11,37 @@ function instrument(options) {
   var instrumenter = new Instrumenter.Instrumenter();
   var captured = false;
 
-  function transform(file) {
-    // If if doesnt match the pattern dont instrument it
+  return through.obj(function(row, enc, next) {
+    if (row.file.indexOf("node_modules") !== -1) {
```

Comment:
I think I understand where the problem is now. Unfortunately, your proposal will only restore the previous behavior if no excludes are specified.

What I’m currently doing is using the `--exclude` option to ignore `.json` and `.mustache` files. If I make the change as you suggest, I would instrument ALL dependencies, including libs like `sinon`. I am then presented what the Sinon coverage my test cases is *haha* - that’s clearly not what I want.

I think to reliably tell the difference between an explicitly included file and an implicit dependency, I would need to check if `row.entry` is `true`. That seems to be the way that Browserify uses internally to tell the difference when transforms are applied. Unfortunately, this is not an official API, but that's best option I can see to achieve the same behavior as before.

I need to run more test with my setup at work, so I can only make the changes on Monday.